enumerate_iam/main.py

```diff
@@ -337,6 +352,13 @@ def enumerate_role(iam_client, output):
         for policy in role_policies['PolicyNames']:
             logger.info('-- Policy "%s"', policy)
 
+            get_policy = iam_client.get_user_policy(RoleName=role_name, PolicyName=policy)
```

Comment:
Maybe do some error handling here? The user might not have permissions to read one of the policies / policy versions, but have permissions to read the next one in `role_policies['PolicyNames']`. We want to handle errors here to extract as much information as possible.